x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -806,6 +811,7 @@ public Settings additionalSettings() {
                 Integer.toString(EsExecutors.allocatedProcessors(settings))
             );
             addMlNodeAttribute(additionalSettings, allocatedProcessorsAttrName, Double.toString(getAllocatedProcessors().count()));
+            addMlNodeAttribute(additionalSettings, mlConfigVersionAttrName, MlConfigVersion.CURRENT.toString());
```

Comment:
We're going to need this attribute on more than just ML nodes. Certainly master and ingest nodes will need it, as some ML operations are performed on master nodes and ingest nodes. But I'm thinking it's best to just add this attribute on all nodes.

The code that determines the min and max versions present in the cluster assumes that the attribute will be set on all nodes.